datasets/ro_sts_parallel/ro_sts_parallel.py

```diff
@@ -0,0 +1,121 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""RO-STS-Parallel: a Parallel English-Romanian Dataset by translating the Semantic Textual Similarity dataset"""
+
+from __future__ import absolute_import, division, print_function
+
+import datasets
+
+
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+Article under review
+"""
+
+# You can copy an official description
+_DESCRIPTION = """\
+The RO-STS-Parallel (a Parallel Romanian English dataset - translation of the Semantic Textual Similarity) contains 17256 sentences in Romanian and English. It is a high-quality translation of the English STS benchmark dataset into Romanian.
+"""
+
+_HOMEPAGE = "https://github.com/dumitrescustefan/RO-STS"
+
+_LICENSE = "CC BY-SA 4.0 License"
+
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "https://github.com/dumitrescustefan/RO-STS/tree/master/dataset/ro-en"
+# _TRAINING_FILE_RO = "RO-STS.train.ro"
+# _TRAINING_FILE_EN = "RO-STS.train.en"
+# _TEST_FILE_RO = "RO-STS.test.ro"
+# _TEST_FILE_EN = "RO-STS.test.en"
+# _DEV_FILE_RO = "RO-STS.dev.ro"
+# _DEV_FILE_EN = "RO-STS.dev.en"
+_DATA_URL = "https://raw.githubusercontent.com/dumitrescustefan/RO-STS/master/dataset/ro-en/RO-STS.{}.{}"
+
+
+class ROSTSParallelConfig(datasets.BuilderConfig):
+    """BuilderConfig for RO-STS-Parallel dataset"""
+
+    def __init__(self, language_pair=(None, None), **kwargs):
+
+        description = ("Translation dataset from %s to %s") % (language_pair[0], language_pair[1])
+        super(ROSTSParallelConfig, self).__init__(description=description, version=datasets.Version("1.0.0"), **kwargs)
+        self.language_pair = language_pair
+
+
+class ROSTSParallel(datasets.GeneratorBasedBuilder):
+    """RO-STS-Parallel dataset"""
+
+    BUILDER_CONFIGS = [
+        ROSTSParallelConfig(
+            name="rosts-parallel-ro-en",
+            language_pair=("ro", "en"),
```

Comment:
Since there's only one language pair, no need to have two configurations (the order doesn't matter)

Reply:
Yes, right, the order doesn't matter, I have kept the version that you suggested and removed the other one. Thanks! I am adding the changes now, with the dataset_infos.json regenerated as well.